src/ToggleButton/toggleButton.css

```diff
@@ -0,0 +1,84 @@
+@import "../proto/base.css";
+
+.default
+{
+  width:              auto;
+  height:             40px;
+  padding:            0 var( --spacing-2 );
+  border:             none;
+  display:            flex;
```

Comment:
Hmm does this `display: flex` work properly in all browsers? 

I remember we had trouble with `flex` on `<button>` elements before: https://stackoverflow.com/questions/35464067/flexbox-not-working-on-button-or-fieldset-elements).